build.cmd

```diff
@@ -834,29 +828,20 @@ if %__BuildPackages% EQU 1 (
 
     echo %__MsgPrefix%Building Packages for %__BuildOS%.%__BuildArch%.%__BuildType%
 
-    set __BuildLogRootName=Nuget
-    set __BuildLog="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-    set __BuildWrn="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
-    set __BuildErr="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
-    set __MsbuildLog=/flp:Verbosity=normal;LogFile=!__BuildLog!
-    set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
-    set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
-    set __Logging=!__MsbuildLog! !__MsbuildWrn! !__MsbuildErr!
+    set __BuildLog="%__LogsDir%\Nuget_%__BuildOS%__%__BuildArch%__%__BuildType%.binlog"
 
     REM The conditions as to what to build are captured in the builds file.
-    call %__ProjectDir%\dotnet.cmd msbuild /nologo /verbosity:minimal /clp:Summary /nodeReuse:false^
-      /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log^
-      /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true^
-      /p:UsePartialNGENOptimization=false /maxcpucount^
-      %__SourceDir%\.nuget\packages.builds^
-      !__Logging! /p:Platform=%__BuildArch% %__CommonMSBuildArgs% %__UnprocessedBuildArgs%
-
+    REM Package build uses the Arcade system and scripts, relying on it to restore required toolsets as part of build
+    powershell -NoProfile -ExecutionPolicy ByPass -NoLogo -File "%~dp0eng\common\build.ps1"^
+        -r -b -projects %__SourceDir%\.nuget\packages.builds^
+        -verbosity minimal /nodeReuse:false /bl:!__BuildLog!^
+        /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true^
+        /p:UsePartialNGENOptimization=false /p:ArcadeBuild=true^
+        !__Logging! /p:Platform=%__BuildArch% %__CommonMSBuildArgs% %__UnprocessedBuildArgs%
```

Comment:
We now have (at least?) two conventions for when we invoke arcade to restore build tasks. `build-job.yml` does it as a separate step, while this does it inside of the build scripts. I think this is fine, just hoping we can converge on one as we onboard more fully onto arcade.

Reply:
Yeah... My thought is that we would move towards everything going through the arcade build scripts (and let them handle the restoring) as we fully onboard onto the entire arcade system.